client/src/utils/string-util.js

```diff
@@ -1,12 +1,34 @@
 import * as moment from 'moment';
 
+/**
+ * Format the specified date into a user-friendly string
+ * @param date - Date to format into a user-friendly string
+ */
 export function prettyPrintDate(date) {
     if (!date) return;
     date = moment(new Date(date));
     // Wednesday, October 16th, 2019 format
     return date.format('dddd, MMMM Do, YYYY');
 }
 
+/**
+ * Parse the specified dateString into a consistent, user-friendly format
+ * @param dateString - String to parse into a consistent, user-friendly format
+ */
+export function prettyPrintDateString(dateString) {
+    if (!dateString) return;
+
+    const dateArray = dateString.split("-");
+    const year = dateArray[0];
+    const month = parseInt(dateArray[1], 10) - 1;
+    const day = dateArray[2];
+
+    const date = moment(new Date(year, month, day));
+
+    // Wednesday, October 16th, 2019 format
+    return date.format('dddd, MMMM Do, YYYY');
```

Comment:
Moment is amazing and will do everything for you

```suggestion
    // Wednesday, October 16th, 2019 format
    const date = moment(dateString).format('dddd, MMMM Do, YYYY');
```

https://momentjs.com/docs/#/parsing/string/ for more info

Reply:
Nice